<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>391</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Dados</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:600;&quot;&gt;CONTROLADOR DE CORRENTE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>221</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(204, 204, 204);</string>
      </property>
      <property name="text">
       <string>           Dados recebidos</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonOnOff_1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>241</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="ImagemMotor1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/electric-motor.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ImagemMotor2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/electric-motor.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Equipamento 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>280</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Equipamento 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>60</y>
        <width>51</width>
        <height>241</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/street-lamp.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Led_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/led-light.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="Led_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/led-light.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableWidget" name="Tabela">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>310</y>
        <width>331</width>
        <height>141</height>
       </rect>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Sensor 1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Sensor 2</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>291</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Sensor 1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="sensor">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>221</width>
        <height>24</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
           </property>
           <property name="text">
            <string>Sensor 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="labelCorrente2"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="menor">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>460</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Porta</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>500</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>velocidade</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>231</width>
        <height>25</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="CONECTAR">
         <property name="text">
          <string>CONECTAR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>291</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Sensor 1</string>
      </property>
      <widget class="QLineEdit" name="Corrente1">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>281</width>
        <height>161</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Sensor 2</string>
      </property>
      <widget class="QLineEdit" name="Corrente2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionCarregar"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCarregar">
   <property name="text">
    <string>Carregar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
